StepDefinition/LanguageFeatureStepDefinitions.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using Project_Mars.Pages;
+using Project_Mars.Utilities;
+using Reqnroll;
+
+namespace Project_Mars.StepDefinition
+{
+    [Binding]
+    
+    public class LanguageFeatureStepDefinitions : CommonDriver
+    {
+        [Given("I logged into Project Mars successfully for language management")]
+        public void GivenILoggedIntoProjectMarsSuccessfullyForLanguageManagement()
+        {
+            driver = new ChromeDriver();
+
+            LoginPage loginPageObj = new LoginPage();
```

Comment:
your creation of objects should be moved to constructor 


Please make use of constructors, please see 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors